release_notes/topics/18_0_0.adoc

```diff
@@ -1,5 +1,22 @@
 = Highlights
 
+== New Operator tech-preview
+
+With this release, we're introducing a brand new {project_operator} as a tech-preview. Apart from being rewritten from
+scratch, the main user-facing change from the legacy Operator is the used {project_name} distribution – the new Operator
+uses the Quarkus distribution of {project_name}. With that, there are a few breaking changes. For details, incl.
+installation and migration instructions, see the https://www.keycloak.org/guides#operator[Operator related guides].
+
+The link:{operatorRepo_link}[legacy Operator] will receive updates until June 2022 when the {project_name} WildFly
+distribution reaches EOL.
+
+=== OperatorHub versioning scheme
+To avoid version conflicts with the legacy Operator, the 18.0.0 version of the new Operator is released as version
+`20.0.0-alpha.1` on OperatorHub. The legacy Operator versioning scheme remains the same, i.e. it is released as 18.0.0.
+
+The same pattern will apply for future {project_name} 18 and 19 releases, until version 20 where the legacy Operator
```

Comment:
Should we reflect the same statement in the main Readme file of `kubernetes-operator` repository?

Reply:
You mean the `description` in CSV? 👍  to that. We can actually sneak in the change in https://github.com/keycloak/keycloak/pull/11247 maybe.

Reply:
I mean here:
https://github.com/keycloak/keycloak-operator/blob/main/README.md

Reply:
Sure. Together with some deprecation note.